packages/venia-concept/local-intercept.js

```diff
@@ -17,6 +17,13 @@
  * or modify functionality from its dependencies.
  */
 
-function localIntercept() {}
-
-module.exports = localIntercept;
+module.exports = (targets) => {
+    targets.of('@magento/venia-ui').routes.tap((routes) => {
+        routes.push({
+            name: 'Comments Page',
+            pattern: '/comments',
+            path: require.resolve('./src/components/CommentsPage'),
```

Comment:
For the future, 
there is also another way to redirect, id additional params here:
        "authed": true,
        "redirectTo": "/"